code/modules/antagonists/heretic/knowledge/sacrifice_knowledge/sacrifice_knowledge.dm

```diff
@@ -81,7 +81,8 @@
 	// Let's remove any humans in our atoms list that aren't a sac target
 	for(var/mob/living/carbon/human/sacrifice in atoms)
 		// If the mob's not in soft crit or worse, or isn't one of the sacrifices, remove it from the list
-		if(sacrifice.stat < SOFT_CRIT || !(sacrifice in heretic_datum.sac_targets))
+		// We specially handle cultist sacrifices and reward the heretic.
+		if(sacrifice.stat < SOFT_CRIT || !(sacrifice in heretic_datum.sac_targets) || !IS_CULTIST(sacrifice))
```

Comment:
This is breaking normal heretic sacrifices during testmerges, you're making it so that _only_ cultists can ever be sacrificed.

Reply:
This also looks likely to be true for several other checks you have added.

Reply:
sad face

Reply:
wtf is 'several otehr checks' man. the other checks in this filwe??

Reply:
Reading over the other ones in this file they're probably fine actually